src/assets/custom/css/_sass/_training-bio.scss

```diff
@@ -0,0 +1,149 @@
+.training-bio {
+  @include lateral-spacing;
+
+  @include medium-large-and-extra-large {
+    @include max-width--out-there;
+  }
+}
+
+.training-bio__inner {
+  @include style-card-box-shadow;
+  display: flex;
+  transition: max-height 0.5s ease;
+
+  .training-bio:not(.training-bio--open) & {
+    @include small {
+      max-height: 590px;
+    }
+    @include medium-large-and-extra-large {
+      max-height: 300px;
+    }
+  }
+
+  .training-bio--open & {
+    @include small {
+      max-height: 1660px;
+    }
+    @include medium {
+      max-height: 830px;
+    }
+    @include large {
+      max-height: 600px;
+    }
+    @include extra-large {
+      max-height: 450px;
+    }
+  }
+
+  @include small {
+    flex-direction: column;
+  }
+  @include medium-large-and-extra-large {
+    flex-direction: row;
+  }
+}
+
+.training-bio__instructor-container {
+  align-items: center;
+  background-color: $navy--ultralight;
+  display: flex;
+  flex-direction: column;
+  padding: 15px 25px;
+}
+
+.training-bio__instructor-avatar {
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: block;
+  height: 180px;
+  margin: 15px;
+  width: 164px;
+}
+
+.training-bio__instructor-title,
+.training-bio__instructor-name {
+  color: black;
+  display: block;
+  margin-left: 6px;
+}
+
+.training-bio__instructor-title {
+  @extend .training-hero__instructor-title;
```

Comment:
We might want to avoid extending styles from a different stylesheet.

Reply:
Amended here https://github.com/codurance/site/pull/1955/commits/ec66d1198e3a8dc93e1fc46a30c81cd154f2679f